test/gui/shared/scripts/pageObjects/EnterPassword.py

```diff
@@ -71,7 +71,7 @@ def get_username(self):
 
     def enter_password(self, password):
         squish.waitForObjectExists(
-            self.PASSWORD_BOX, get_config("maxSyncTimeout") * 1000
+            self.PASSWORD_BOX, get_config("minSyncTimeout") * 1000
```

Comment:
this might be the cause for drone timeout. fucntion `waitForObjectExists` fails to get the password object and waits till timeout i.e. 60 seconds. so if that happens in most of the scenarios then the time is more than 1 hour.

Reply:
now timeout set to only 5 seconds